tests/modules/correct-module.js

```diff
@@ -90,6 +90,13 @@ module.exports = async function plugin(fastify, config) {
     }, 'error logs')
     reply.send({})
   })
+
+  fastify.get('/with-audit-logs', function handler(request, reply) {
+    this.log.audit({
```

Comment:
What if used with typescript service?

Reply:
You are right, I updated the declaration file accordingly.